packages/contracts-bedrock/test/L2/L1BlockInterop.t.sol

```diff
@@ -1,52 +1,209 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.15;
 
-import { Test } from "forge-std/Test.sol";
-import { Encoding } from "src/libraries/Encoding.sol";
-import { L1BlockInterop, DependencySetSizeMismatch, NotDepositor, ConfigType } from "src/L2/L1BlockInterop.sol";
+// Testing utilities
+import { CommonTest } from "test/setup/CommonTest.sol";
 
-contract L1BlockInteropTest is Test {
-    L1BlockInterop l1Block;
-    address depositor;
+// Target contract dependencies
+import { L1BlockInterop, ConfigType } from "src/L2/L1BlockInterop.sol";
 
-    function setUp() public {
-        l1Block = new L1BlockInterop();
-        depositor = l1Block.DEPOSITOR_ACCOUNT();
+contract L1BlockInteropTest is CommonTest {
+    /// @notice Thrown when a non-depositor account attempts to set L1 block values.
+    error NotDepositor();
+
+    /// @notice Error when a chain ID is not in the interop dependency set.
+    error NotDependency();
```

Comment:
We should not define these events twice and just have a single source of truth

Reply:
we could define them in `L1Block.sol`, but it would require moving the errors and events to _outside_ the contract (but in the same file) so that they can be imported by the test file. some errors are defined inside the original `L1Block` inherited from `L1BlockInterop`. should we redefine those errors? leaning slightly against modifying `L1Block`

Reply:
One solution that clabby uses is having a file with only errors defined in it and then importing that file in multiple files, see https://github.com/ethereum-optimism/optimism/blob/13d614684ad4395374367944c8211976fac85578/packages/contracts-bedrock/src/dispute/DisputeGameFactory.sol#L12

Reply:
makes sense! this is similar to the current approach with `OptimismPortal`